eth/db/journal.py

```diff
@@ -262,3 +263,27 @@ def reset(self) -> None:
         """
         self.journal = Journal()
         self.record()
+
+    def diff(self) -> DBDiff:
+        """
+        Generate a DBDiff of all pending changes.
+        These are the changes that would occur if you called :meth:`persist()`.
```

Comment:
*Nitpick*, We try to avoid *you* form, so this should rather be:

>These are the changes that would occur if one called :meth:`persist()`.

Or maybe 

>These are the changes that would occur :meth:`persist()` was called